### PR TITLE
add cinema ticket files to repository

### PR DESCRIPTION
**Pull request by olalipa**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <div class="category">VIP</div>
+        <div class="price">$10</div>
+      </div>
+    </div>
+    <div class="seance-details">
+      <div class="hour">08:00 PM
+      </div>
+      <div class="cinema-room-details">
+        <div class="gate">GATE: 03</div>
+        <div class="seat"> SEAT: B43</div>
```
Comment (Solnick) [inline index.html]:
```suggestion
        <div class="seat">SEAT: B43</div>
```
Don't add redundant spaces

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      </div>
+      <div class="ticket-number">TICKET NO: 103752
+      </div>
```
Comment (Solnick) [inline index.html]:
```suggestion
      <div class="ticket-number">TICKET NO: 103752</div>
```
This new line is not needed

File: style.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
+.ticket-container {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+    font-family: Roboto;
```
Comment (Solnick) [inline style.css]: remember about adding fallback font

File: style.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    height: 128px;
+    font-size: 55px;
+    margin-top: 26px;
+    margin-left: 20px;
+}
+
+.ticket-details {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 63px;
```
Comment (Solnick) [inline style.css]: When using margin or padding make sure that the values ar multiplication of the same number, usually it is a multiplication of 4px. It makes elements positioning and aligning a lot easier

**Review by Solnick — CHANGES_REQUESTED** [review]
image
I think the font sizes are different. Please check in figma what font size to use.

File: style.css (hunk 4 above)
Comment (Solnick) [inline style.css]: Using such a big margin value is always a red flag. Probably you can achieve the same effect by making the parent component bigger and setting up the flexbox align/justify-content.

**Review by Solnick — CHANGES_REQUESTED** [review]

**Review by Solnick — CHANGES_REQUESTED** [review]
Please change the code according to the comments. You ignored the comments about: 1. big padding and margins
2. spacing should be multiplication of 4px

**Comment by olalipa** [comment]
> Please change the code according to the comments. You ignored the comments about: 1. big padding and margins 2. spacing should be multiplication of 4px

I am not sure if I get you comment correctly.. I already changed big margin (63 px) last time. Regarding spacing that should be multiplication of 4px - why is 20 px incorrent in this case?

**Review by Solnick — APPROVED** [review]

**Pull request merged**